anime_downloader/sites/animesvostfr.py

```diff
@@ -0,0 +1,262 @@
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+import logging
+import re
+
+from requests.exceptions import HTTPError
+from anime_downloader.config import Config
+from anime_downloader.extractors import get_extractor
+
+logger = logging.getLogger(__name__)
+
+class AnimesVOSTFR(Anime, sitename='animesvostfr'):
+       
+        sitename = 'animesvostfr'
+        QUALITIES = ['360p', '480p', '720p', '1080p']
+        DOMAIN = "https://www1.animesvostfr.net/"
+
+        @classmethod
+        def search(cls, query):
+            
+            soup = helpers.soupify(helpers.get(cls.DOMAIN, params={'s' : query}))
+            
+            results = []
+            
+            for v in soup.select('div.movies-list > div.ml-item'):
+                t = v.h2.text
+                u = v.a['href']
+                p = v.img['src']
+                
+                lang = v.select_one('div.language').text.upper()
+                
+                #logger.debug( "results: {}".format(lang) )
+                
+                eps = v.select_one('span.mli-quality').text.upper()
+                
+                #logger.debug( "results: {}".format(eps) )
+                
+                lang_dict = {
+                'VOSTFR':f' (VOSTFR {eps})',
+                'VF':f' (VF {eps})',
+                'VO':f' (VO {eps})'
+                }
+                selected_lang = f' (V? {eps})' #default lang
+                for a in lang_dict:
+                    if a in lang.upper():
+                        selected_lang = lang_dict[a]
+                        break
+                t += selected_lang
+                
+                #logger.debug( "title: %s", t )
+ 
+                search_result_info = SearchResult(title=t, url=u, poster=p)
+               
+                results.append(search_result_info)
+
+            
+            logger.debug( "results: {}".format(results) )
+            
+            return results
+        
+        def _scrape_episodes(self):
+            soup = helpers.soupify(helpers.get(self.url))
+
+            episode_links = None
+            results = [] 
+            
+            season = soup.find("div", id="seasonss")
+            
+            if season:
+                episode_links = season.select('div.les-title > a')
+                
+            
```

Comment:
Don't leave so many unnecessary spaces

Reply:
which spaces ?
Question : what is the difference with season.select('div.les-title > a') and season.select('div.les-title a') ?

Reply:
> which spaces ?

Spaces between everything you https://github.com/vn-ki/anime-downloader/blob/882b788066c4fcdbfb344d1fb0b005d55134b71b/anime_downloader/sites/animesvostfr.py#L53-L54 You shouldn't have 2 spaces (blank lines) between something in the same function

Also take a look at https://www.w3schools.com/cssref/css_selectors.asp